idunn/services/instant_answer/normalization.py

```diff
@@ -8,7 +8,16 @@
     ignore_pattern_end = re.compile(rf' +({"|".join(patterns)})$')
 
 
-def normalize(query):
+def normalize_instant_answer_param(query, user_country) -> (str, str):
+    query = query.strip().lower()
+    query = ignore_pattern_start.sub("", query)
+    query = ignore_pattern_end.sub("", query)
+    if user_country is not None:
+        user_country = user_country.lower()
+    return query, user_country
+
+
+def normalize_search(query) -> str:
     query = query.strip().lower()
```

Comment:
Feels like `idunn.services.**instant_answer**` is not the place then? :/

How would you define what a service is? Maybe we just shouldn't have an `instant_answer` submodule?

Reply:
I totally give up the service layer in this PR. I think I forgot to undo this move. (I think it will be easier to do it on TS migration)